pom.xml

```diff
@@ -102,7 +102,7 @@
             <dependency>
                 <groupId>javax.validation</groupId>
                 <artifactId>validation-api</artifactId>
-                <version>2.0.1.Final</version>
+                <version>1.1.0.Final</version>
```

Comment:
❓ Why does this have to be the same as DDFs version?

Reply:
The 1.1.0.Final bundle is already started in DDF 2.21.0: https://github.com/codice/ddf/blob/e25fae622f175ea3991d8afee07757c6a2f23d38/features/kernel/src/main/feature/feature.xml#L80-L82. If the version does not match, then the validation-api needs to be embedded in the IonSource bundle or started up in the ion-source feature. I thought updating the version to match DDF is the preferred solution.

Reply:
I don't think we should make this repo dependent on DDF in any way.

Reply:
See https://github.com/connexta/ion-query-api/pull/41.